app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesViewImpl.kt

```diff
@@ -40,7 +46,17 @@ class AllNotesViewImpl(bundle: Bundle) : BaseViewImpl<AllNotesView, AllNotesPres
 
     override fun showSingleNote(id: String?) {
         Timber.d("Request to display Note $id")
+
+        var args: Bundle = makeNoteDetailBundle(id)
+        router.pushController(RouterTransaction.with(NoteDetailViewImpl(args)))
     }
 
     override fun addNoteClicks(): Observable<Unit> = addNote.clicks()
+
+    private fun makeNoteDetailBundle(id: String?) : Bundle {
+        val args: Bundle = Bundle()
+        args.putString(NoteDetailViewImpl.Constants.ARG_NOTE_ID, id)
```

Comment:
this should be a new method on screenBundleHelper.  The idea is to isolate all of these strings to a single class and privatize them, with getters and setters for each.